lib/pbench/server/api/resources/upload_api.py

```diff
@@ -48,85 +32,6 @@ def __init__(self, status: int, message: str):
         self.message = message
 
 
-class CleanupAction:
```

Comment:
Why does the removal of this code from the server side not result in other server side changes?

Reply:
It works exactly the same with the addition of the import to line 11.

Reply:
Great, this is the only module that actually used it.  Thanks for confirming.